app/src/components/widgets/charts/failedCasesTrendChart/configuration.js

```diff
@@ -0,0 +1,49 @@
+const range = (start = 0, stop, step = 1) => {
```

Comment:
This should be moved to utils.
This function needs tests. Here's my idea for a set of tests. Feel free to extend it (I think edge cases should be covered too, for example, range below zero, etc...).
```
describe('range', () => {
  test('it should return an array');
  test('it should return an empty array when invoked with no arguments');
  test('it should return an array with correct length');
  test('it should return an array of numbers increasing on step');
});
```